check-eve.py

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env python3
+
```

Comment:
This needs a license and copyright. You can copy it from `run.py`, but use a date of just 2021.